core/db/migrate/20161012151358_opening_ledger_entries_for_store_credits.rb

```diff
@@ -0,0 +1,9 @@
+class OpeningLedgerEntriesForStoreCredits < ActiveRecord::Migration[5.0]
+  def up
+    Rake::Task["solidus:migrations:create_ledger_entries_for_store_credits:up"].invoke
```

Comment:
I wanted to have a way to properly spec it, could not find a better way to do this.